dissect/hypervisor/descriptor/vbox.py

```diff
@@ -11,7 +11,8 @@ def __init__(self, fh: IO):
         self._xml: Element = ElementTree.fromstring(fh.read())
 
     def disks(self) -> Iterator[str]:
-        for hdd_elem in self._xml.findall(
-            f".//{self.VBOX_XML_NAMESPACE}HardDisk[@location][@format='VDI'][@type='Normal']"
-        ):
-            yield hdd_elem.attrib["location"]
+        for hdd_elem in self._xml.findall(f".//{self.VBOX_XML_NAMESPACE}HardDisk[@location][@type='Normal']"):
```

Comment:
Does the spec specify if `Normal` is case sensitive?

Reply:
Yes, Normal is case sensitive: https://www.virtualbox.org/browser/vbox/trunk/src/VBox/Main/xml/VirtualBox-settings.xsd#L139